### PR TITLE
Update our-workflow.md

### PR DESCRIPTION
**Pull request by kniz-raft**
### Summary of Changes
_Updates documentation as a result of ADR 009 and new team members being added_
_Addresses issue #700_

### CODE REVIEW

File: docs/How-We-Work/team-charter/our-workflow.md [hunk 1]
```diff
@@ -53,7 +54,7 @@ Based on the September 2020 iron triangle conversations, the product team commit
 ## Work on the issue is started
 - When the individual assigned to work on an issue starts work on it, they should move it to the “In Progress” column/pipeline.
 - Any decisions or discussions about the issue should be documented within the issue
-- Ad hoc pairing or scheduled pairing session(s) with 18F are encouraged prior to the Day 13 QASP review so the 18F team does not see progress on the issue for the first time on Day 13.
+- Ad hoc pairing or scheduled pairing session(s) with 18F are encouraged throughout the sprint to increase transparency and familiarity with the issue
```
Comment (carltonsmith) [inline docs/How-We-Work/team-charter/our-workflow.md:57]: I would say
> ... with 18F and ACF Tech Lead...

**Review by carltonsmith — CHANGES_REQUESTED** [review]
One small change, but looks great!

File: docs/How-We-Work/team-charter/our-workflow.md (hunk 1 above)
Comment (kniz-raft) [reply docs/How-We-Work/team-charter/our-workflow.md]: Good catch - updated. Thanks!

**Review by carltonsmith — APPROVED** [review]
LGTM!

**Review by ADPennington — COMMENTED** [review]
lgtm. general question: do we need to call out the security control-related workflow? It is reflected in the ADR#9 which is referenced here, so maybe that's sufficient?

**Comment by lfrohlich** [comment]
> lgtm. general question: do we need to call out the security control-related workflow? It is reflected in the ADR#9 which is referenced here, so maybe that's sufficient?

That's a good point. Maybe we could say something like "see ADR#9 including security control-related workflow"

**Comment by lfrohlich** [comment]
Looks good -- one suggestions that "The product owner proposes the issues for the next sprint" could be expanded to say issues come from PM/leads/Tech Lead, and PO.

**Review by lfrohlich — APPROVED** [review]
THanks for making that change!

**Comment by kniz-raft** [comment]
> THanks for making that change!

Sure thing - @ADPennington I added more context around ADR #009 too if you want to take a look.

**Review by ADPennington — APPROVED** [review]
Looks good! Thanks @kniz-raft

**Pull request merged**